ui-src/src/components/MiniMap.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+
+import {Action, ActionType, GameBoard, MoveDefinition, XY} from '../../../minesweeper'
+
+import './Cell.css';
+
+import {Hash} from '../../../holochain';
+
+import CellMatrix from '../CellMatrix';
+import {fetchJSON} from '../common'
+import store from '../store'
+
+interface IminiMapProps {
```

Comment:
I added a tslint rule that interfaces cannot start with "I" (the alternative is that they ALL must start with "I"), did you try "building" (yarn starting) this?

Reply:
linter seems happy and I can build and run. I actually didn't mean to include this file though I guess I am getting sloppy. Was supposed to be back-end only changes.